.gitignore

```diff
@@ -20,7 +20,7 @@ windows/plugins
 opentelemetry-java-contrib-jmx-metrics.jar
 VERSION.txt
 release_deps
-tmp
+/tmp
```

Comment:
curious why we need the leading `/`?

Reply:
I have a testdata directory that has a tmp directory (it's the in the folder structure we are using for updating), so I needed to change something to make sure git picks it up.